framework/DataObjects/DataSet.py

```diff
@@ -2065,18 +2066,19 @@ def _toCSVXML(self,fileName,**kwargs):
       toRemove = []
       ## TODO doesn't work for time-dependent requests!
       genNode =  xmlUtils.findPath(meta['DataSet'].getRoot(),'general')
-      for child in genNode:
-        if child.tag in ['inputs','outputs','pointwise_meta']:
-          vs = []
-          for var in child.text.split(','):
-            if var.strip() in keep:
-              vs.append(var)
-          if len(vs) == 0:
-            toRemove.append(child)
-          else:
-            child.text = ','.join(vs)
-      for r in toRemove:
-        genNode.remove(r)
+      if genNode is not None:
```

Comment:
what is the rationale behind this line being added here?

Reply:
@wangcj05 ?

Reply:
This is just to make sure the point wise meta is presented in the dataset.